src/views/UserSearch/Filters.js

```diff
@@ -1,38 +1,116 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  get,
+  sortBy,
+} from 'lodash';
 
 import {
-  FilterGroups,
+  Accordion,
+  AccordionSet,
+  FilterAccordionHeader,
 } from '@folio/stripes/components';
 
+import {
+  CheckboxFilter,
+  MultiSelectionFilter,
+} from '@folio/stripes/smart-components';
+
+import { statusFilter } from '../../constants';
+
 export default class Filters extends React.Component {
   static propTypes = {
     activeFilters: PropTypes.object,
+    resources: PropTypes.object.isRequired,
     onChangeHandlers: PropTypes.object.isRequired,
-    config: PropTypes.arrayOf(PropTypes.object),
   };
 
   static defaultProps = {
     activeFilters: {},
   }
 
-  render() {
+  getStaticFilterValues = (items) => {
+    return items.map(item => ({ label: item, value: item }));
+  };
+
```

Comment:
We should convert `label` to a translated value, i.e. a `<FormattedMessage>` corresponding to something with a key like `filters.statusFilter.${item}`, since these are static values. This isn't criticism; the previous code didn't handle the situation either. Seeing `getStaticFilterValues()` nicely factored out just highlights the fact that the translation is missing. Filed [UIU-1467](https://issues.folio.org/browse/UIU-1467).

Reply:
Is it okay if I will do this in scope of this user story?

Reply:
Up to you. I'd probably do it separately just to avoid having this PR languish if that part turns out to be more complicated than you expect. But if it turns out to be simple and you can sneak it in, I'm not gonna argue!